interfaces/Portalicious/src/app/pages/project-payments/components/create-payment/create-payment.component.html

```diff
@@ -0,0 +1,61 @@
+<p-button
+  label="Create new payment"
+  i18n-label="@@create-payment"
+  rounded
+  icon="pi pi-money-bill"
+  (click)="dialogVisible.set(true); createPaymentDialog.maximize()"
+/>
+
+<p-dialog
+  #createPaymentDialog
+  [(visible)]="dialogVisible"
+  [draggable]="false"
+  [resizable]="false"
+  [modal]="true"
+  styleClass="bg-grey-100 [&>*]:bg-grey-100 [&_.p-dialog-footer]:p-0 [&_.p-dialog-footer]:shadow"
+>
+  <ng-template pTemplate="header">
+    <h3>
+      <i class="pi pi-money-bill me-2"></i>
+      <ng-container i18n="@@create-payment">Create new payment</ng-container>
+      - {{ today | date: 'shortDate' }}
+    </h3>
+  </ng-template>
+  <div class="w-full px-6 lg:px-24">
+    @defer (when dialogVisible()) {
+      <p
+        i18n
+        class="mb-5"
+      >
+        Select included registrations to add to this payment.
+      </p>
+      <p-card>
+        <!-- TODO AB#31329 limit this to included registrations -->
+        <app-registrations-table
+          #registrationsTable
+          [projectId]="projectId()"
+          [localStorageKey]="
+            'project-payment-registrations-table-' + projectId()
+          "
+        />
+      </p-card>
+    }
+  </div>
+  <ng-template pTemplate="footer">
+    <div class="w-1/2 border-t-2 border-t-purple-300"></div>
+    <div class="flex items-center justify-end p-4">
+      <span
+        i18n
+        class="absolute left-1/2 -translate-x-1/2"
```

Comment:
Does `start-1/2` also work?

And would there be an equivalent for `translate-x`?

Reply:
I couldn't find an equivalent for `translate-x`. That said, given that we're centering the item, it wouldn't make a difference. It could be `right-1/2 translate-x-1/2` and it would be the same result.

Reply:
In this case it is "centering in relation to its own width" right?

Then it won't matter, as its parent container would align properly because of the `justify-end`-class, I'd think.

Reply:
I tried both of those already. The first option is differnt from the design. The second option doesn't work:

![image](https://github.com/user-attachments/assets/186835c2-36cb-4fdb-b6e9-7a57a13c73ce)

For now I'm leaving it as-is.